shopinvader_restapi/views/shopinvader_menu.xml

```diff
@@ -22,13 +22,6 @@
         groups="sales_team.group_sale_salesman"
```

Comment:
Don't forget to change web_icon path line 13 of this file (sorry but GitHub doesn't let me point out a line with no change):

`web_icon="shopinvader,static/description/icon.png"` -> `web_icon="shopinvader_restapi,static/description/icon.png"`